.github/workflows/ci-golang-lint.yml

```diff
@@ -15,8 +15,7 @@ on:
     - cron: "0 * * * *"
 jobs:
   golang-lint:
-    # TODO: uncomment this after release work
-    # if: github.event.pull_request.draft == false
+    if: github.event.pull_request.draft == false
```

Comment:
@adityasadalage I think this will need fixing for other yaml as well.

Reply:
Noted